### PR TITLE
Added Chinese Font ‘PangMenZhengDao Heading’

### PR DESCRIPTION
**Pull request by XHXIAIEIN**
This font is made by PangMenZhengDao and is free for use.

### CODE REVIEW

File: ofl/pangmenzhengheading/METADATA.pb.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+category: "SANS_SERIF"
+date_added: "2020-04-08"
+fonts {
+  name: "PangMenZhengDao Heading"
+  style: "normal"
+  weight: 400
+  filename: "PangMenZhengDaoHeading-Regular.ttf"
+  post_script_name: "PangMenZhengDao-Heading"
+  full_name: "PangMenZhengDao Heading"
+  copyright: "Copyright (c) 2016 by PangMenZhengDao and HuXIaoBo, with Reserved Font Name \'PangMenZhengDao Heading\'"
```
Comment (davelab6) [inline ofl/pangmenzhengheading/METADATA.pb.yml:13]: We don't accept RFNs

File: ofl/pangmenzhengheading/METADATA.pb.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
+name: "PangMenZhengDao Heading"
+designer: "PangMenZhengDao"
+license: "OFL"
+category: "SANS_SERIF"
+date_added: "2020-04-08"
```
Comment (davelab6) [inline ofl/pangmenzhengheading/METADATA.pb.yml:5]: `date_added: "2022-01-08"` or so

**Comment by davelab6** [comment]
Thanks @XHXIAIEIN for submitting this. I added a few quick comments, and we'll take a closer look next year :)

File: ofl/pangmenzhengheading/DESCRIPTION.en_us.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<p>庞门正道标题体，由设计公众号"<a href="https://mp.weixin.qq.com/mp/profile_ext?action=home&__biz=MzA5NjIzNjgxNw==&scene=124#wechat_redirect">《庞门正道》</a>"以及13位字体设计师联合研发。</p>
```
Comment (davelab6) [inline ofl/pangmenzhengheading/DESCRIPTION.en_us.html:1]: We also need an English description

**Comment by XHXIAIEIN** [comment]
> Thanks @XHXIAIEIN for submitting this. I added a few quick comments, and we'll take a closer look next year :)

Oh Sorry. I am not sure about the uploading rules, so I will re-read the instructions carefully.Thanks for your review. Is there anything else I need to do?

Because the author of the font has several open fonts to upload.

File: ofl/pangmenzhengheading/DESCRIPTION.en_us.html (hunk 3 above)
Comment (XHXIAIEIN) [reply ofl/pangmenzhengheading/DESCRIPTION.en_us.html:1]: OK, I will resubmit as soon as possible.

**Comment by XHXIAIEIN** [comment]
I have also uploaded several other fonts, please check again.

**Comment by RosaWagner** [comment]
@XHXIAIEIN thanks for the update, but let's not go too far too fast ^^ 

I would rather focus on having one font onboarded before proceeding with several fonts (in separated PRs later if we manage to have one). Indeed, there are several things that need to be corrected in this PR and the sources themselves, and someone of GF team will have to take a look next year.

Most of all, we need the URL of the git repo where the fonts are hosted _with the sources_. Since you are not the original designer, we would also need some kind of proof of his agreement: only the copyright holders can decide to remove the Reserved Font Name from the Licence — which is missing from this PR.

Please agree with all of that:

- [ ] The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
- [ ] There is no "Reserved Font Name" mention in the OFL license
- [ ] Family name is unique according to namecheck.fontdata.com
- [ ] If the font targets latin based written languages, it supports at least [GF Latin Core glyphset](https://github.com/googlefonts/gftools/blob/main/Lib/gftools/encodings/GF%20Glyph%20Sets/GF-latin-core_unique-glyphs.nam)
- [ ] I read and agree with the full [contributing requirements](https://github.com/google/fonts/blob/main/CONTRIBUTING.md)
- [ ] The repository respect the [upstream repo structure](https://github.com/googlefonts/gf-docs/tree/main/Spec#upstream-repo-structure)
- [ ] The authors are actively maintaining the source repository

Also, read this documentation. It is not super updated, we are working on improving it, but it will do for now:
https://github.com/googlefonts/gf-docs/tree/main/Spec

**Comment by XHXIAIEIN** [comment]
license:   
https://mp.weixin.qq.com/s/rD9WfcVUXblQXG8FF7dqZg


> 庞门正道的字体全部都是商用免费的！  
> All of PangMenZhengDao fonts are free for commercial use!

list:   
- 庞门正道标题体 ( PangMenZhengDao  `BiaoTi ` /  `Heading` )
- 庞门正道细线体 (PangMenZhengDao  `XiXian` /  `Line` )
- 庞门正道轻松体 (PangMenZhengDao  `QingSong` / `Paper` )
- 庞门正道粗书体 (PangMenZhengDao  `Cushu Kai` )
- 庞门正道真贵楷体 (PangMenZhengDao  `Zhengui Kai` )

**Comment by aaronbell** [comment]
Hi @XHXIAIEIN, thanks for submitting these! I'm happy to help support their onboarding to Google Fonts.

I took a look at the font files themselves and found that all of the fonts will require some degree of fixes—some to a greater extent than others—in order to meet Google's standards for release. As such, these are not viable to merge at this time. We will need to rebuild / update the source and rebuild the fonts. 

Per @RosaWagner's comment, these fonts will need to be uploaded to a repository where we can store the sources and where they can be worked on (rather than trying to do everything in this PR). 

In terms of Rosalie's checklist, I can assist with updating the glyphset and getting the repository structure set up correctly. However, everything else needs to be addressed before we can get started. 

Thanks!

**Comment by RosaWagner** [comment]
@XHXIAIEIN The fonts does not match our requirements so far and without news from you I'd rather close this PR. 

Please feel free to open an issue in https://github.com/google/fonts/issues to request the addition of these families if it moves forward (for example, if the original designer agrees to the requirements or contact us directly to work towards that goal). We would be happy to help.

The license would be the first step, it is not enough that it guarantees free commercial use, it also has to be the [OFL from SIL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL), such as most of Google Fonts' fonts (see https://github.com/google/fonts/blob/main/ofl/asap/OFL.txt as an example).

**Pull request closed without merging**